### PR TITLE
Setup TypeScript & ESLint

### PR DESCRIPTION
**Pull request by cometkim**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,10 +43,16 @@
       "eslint --fix"
     ]
   },
+  "files": [
+    "gatsby-node.js",
+    "gatsby-ssr.js",
+    "utils.js",
+    "LICENSE",
+    "README"
+  ],
```
[the remark this answers is not in the document]
Comment (cometkim) [reply package.json:52]:
```suggestion
  "files": [
    "index.js",
    "gatsby-node.js",
    "gatsby-ssr.js",
    "utils.js",
    "LICENSE",
    "README"
  ],
```

File: .eslintrc.js [hunk 2]
```diff
@@ -1,18 +1,14 @@
 module.exports = {
-  parser: 'babel-eslint',
+  plugins: [
+    '@cometjs',
+  ],
```
[the remark this answers is not in the document]
Comment (cometkim) [reply .eslintrc.js:4]:
```suggestion
```

[the remark this answers is not in the document]
Comment (cometkim) [reply .eslintrc.js]: Thanks @silvenon I didn't realize that's not necessary 😂

**Comment by silvenon** [comment]
Let me know when it's ready for another review.

**Pull request merged**